src/KeyedFrames.jl

```diff
@@ -174,17 +174,22 @@ function DataFrames.deleterows!(kf::KeyedFrame, ind)
     return kf
 end
 
-@deprecate deletecols!(kf::KeyedFrame, ind) select!(kf, ind)
+@deprecate deletecols!(kf::KeyedFrame, ind) select!(kf, Not(ind))
```

Comment:
This does work for multiple columns which I wasn't sure about. May be worth changing this to:
```suggestion
@deprecate deletecols!(kf::KeyedFrame, inds) select!(kf, Not(inds))
```